### ISSUE TITLE
Refactor all hard-coded project names to use config variable

### ISSUE THREAD

**Issue by bellerbrock** [solgenomics/sgn#2735]
Expected Behavior 
--------------------------------------------------------------------------

Some files, for example [local_head.mas](https://github.com/solgenomics/sgn/blob/bdcc83e32fc532be520c2cea8c07ddb54c51732e/mason/site/header/local_head.mas#L3), still have the 'Sol Genomics Network' project name hardcoded. It would be better to retrieve the project name whereever it is used from the conf file so that it can easily be changed.

For Bugs:
---------

**Comment by lukasmueller** [comment]
Yes, there are already config keys, but they are not used systematically.
